podman_compose.py

```diff
@@ -2931,13 +2932,15 @@ async def compose_down(compose: PodmanCompose, args):
                         f"label=io.podman.compose.project={compose.project_name}",
                         "-a",
                         "--format",
-                        "{{ .Names }}",
+                        "{{ .Image }} {{ .Names }}",
                     ],
                 )
             )
             .decode("utf-8")
             .splitlines()
         )
+        orphaned_images = [item.split()[0] for item in orphaned_containers]
+        names = {item.split()[1] for item in orphaned_containers}
```

Comment:
Not sure if you really wanted to have set for names but not for orphaned_images. Needs a comment why.

Reply:
Good catch, thanks!